app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+
+    people: dict = {}
```

Comment:
It is not a good practice to use mutable types like `dict` as class-level attributes. This is because they are shared across all instances of the class, which can lead to unexpected behavior. Instead, consider initializing such attributes within the `__init__` method.